docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -0,0 +1,518 @@
+
+.Prerequisites
+
+* 4 vCPUs, 8 GiB RAM, and 40 GB persistent storage.
+* OpenShift Container Platform 4.13-4.15 installed.
+* You must be logged in as a user with `cluster-admin` permissions.
+
+For more information, see link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles].
+
+.Procedure
+
+. In the OpenShift Container Platform web console, click *Operators → OperatorHub*.
+. Use the *Filter by keyword* field to search for *MTA*.
+. Click the *Migration Toolkit for Applications* Operator and then click *Install*.
+. On the *Install Operator* page, click *Install*.
+. Click *Operators → Installed Operators* to verify that the {ProductShortName} Operator appears in the `openshift-mta` project with the status `Succeeded`.
+. Click the *{ProductShortName}* Operator.
+. Under *Provided APIs*, locate *Tackle*, and click *Create Instance*.
++
+The *Create Tackle* window opens in *Form* view.
+. Review the custom resource (CR) settings. The default choices should be acceptable, but make sure to check the system requirements for storage, memory, and cores.
+. To work directly with the YAML file, click *YAML* view and review the CR settings that are listed in the `spec` section of the YAML file.
++
+The most commonly used CR settings are listed in this table:
++
+.Tackle CR settings
+[cols="40%,15%,55%", options="header"]
+|====
+|Name
+|Default
+|Description
+
+|`cache_data_volume_size`
+|`100 GiB`
+|Size requested for the cache volume; ignored when `rwx_supported=false`
+
+|`cache_storage_class`
+|Default storage class
+|Storage class used for the cache volume; ignored when `rwx_supported=false`
+
+|`feature_auth_required`
+|`True`
+|Flag to indicate whether keycloak authorization is required (single user/"`noauth`")
+
+|`feature_isolate_namespace`
+|`True`
+|Flag to indicate whether namespace isolation using network policies is enabled
+
+|`hub_database_volume_size`
+|`5 GiB`
+|Size requested for the Hub database volume
+
+|`hub_bucket_volume_size`
+|`100 GiB`
+|Size requested for the Hub bucket volume
+
+|`hub_bucket_storage_class`
+|Default storage class
+|Storage class used for the bucket volume
+
+|`keycloak_database_data_volume_size`
+|`1 GiB`
+|Size requested for the Keycloak database volume
+
+|`pathfinder_database_data_volume_size`
+|`1 GiB`
+|Size requested for the Pathfinder database volume
+
+|`maven_data_volume_size`
+|`100 GiB`
+|Size requested for the Maven m2 cache volume; deprecated in {ProductShortName} 6.0.1
+
+|`rwx_storage_class`
+|NA
+|Storage class requested for the Tackle RWX volumes; deprecated in {ProductShortName} 6.0.1
+
+|`rwx_supported`
+|`True`
+|Flag to indicate whether the cluster storage supports RWX mode
+
+|`rwo_storage_class`
+|NA
+|Storage class requested for the Tackle RW0 volumes
+
+|`rhsso_external_access`
+|`False`
+|Flag to indicate whether a dedicated route is created to access the {ProductShortName} managed RHSSO instance
+
+|`windup_container_limits_cpu`
```

Comment:
Find replace all references to `windup` with `analyzer` so `analyzer container_limits_cpu`

Reply:
updated